c10/test/util/bfloat16_test.cpp

```diff
@@ -1,4 +1,7 @@
+// clang-format off
 #include <c10/util/BFloat16.h>
+#include <c10/util/BFloat16-math.h>
```

Comment:
Order of imports matter (but clang-format wants `BFloat16-math` as the first include.

(Including only `BFloat16-math.h` would be sufficient)